content/guides/outliers.md

```diff
@@ -80,75 +80,51 @@ There are two different outlier detection algorithms you can use on your data: D
 
 ### DBSCAN
 
-[DBSCAN](https://en.wikipedia.org/wiki/DBSCAN) is a popular density-based clustering algorithm that greedily creates clusters by agglomerating points that are close to each other. Clusters with few points in them are considered outliers.
+A natural way to group together hosts that are behaving similarly is to use a clustering algorithm. We use [DBSCAN](https://en.wikipedia.org/wiki/DBSCAN), a popular density-based clustering algorithm, for this purpose. DBSCAN works by greedily agglomerating points that are close to each other. Clusters with few points in them are considered outliers.
 
-We use a simplified form of DBSCAN to detect outliers; any point that is not in the largest cluster will be considered an outlier. We consider each host to be a d-dimensional vector, where d is the number of values in the time series. Any hosts that are within a distance of `alpha x threshold` of each other will get clustered together.
+Traditionally, DBSCAN takes: 1) a parameter 𝜀 that specifies a distance threshold under which two points are considered to be close; and 2) the minimum number of points that have to be within a point’s 𝜀-radius before that point can start agglomerating. The image below shows an example of DBSCAN in action on points in the plane. There are two clusters. The large points had enough close neighbors to agglomerate those points, while the small colored points did no agglomerating themselves but are within the 𝜀-radius of a large point. The points in black are the outliers.
 
-The threshold is calculated as follows. We create a new median time series by taking the median of the values from the existing time series at every time point. Then we calculate the distance between each host and the median series. The threshold is the median of those distances.
-
-In pseudocode:
-
-~~~ python
-    median_series = pointwise_median(series_list)
-    dists = [dist(series,median_series) for series in series_list]
-    threshold = median(dists)
-~~~
+<img src="/static/images/outliers/outliers-dbscan-2d.png" style="width:100%; border:1px solid #777777"/>
```

Comment:
Not sure if it's different where this is going to be published, but GitHub doesn't like this path to the image. I think `/content` would need to be prepended to the path for this to work.